Project/Assets/ML-Agents/Examples/SharedAssets/Scripts/ProjectSettingsOverrides.cs

```diff
@@ -45,7 +45,7 @@ public void Awake()
 
             // Make sure the Academy singleton is initialized first, since it will create the SideChannels.
```

Comment:
You can remove `var academy = Academy.Instance` (and the comment above) now that we're going through the Academy to get the EnvironmentParameters

Reply:
Good catch.